File: JavaScript/8js 1/contamination.js
/*
|--------------------------------------------------------------------------
| Contamination #1 -String-
|--------------------------------------------------------------------------
|
*/

function contamination(text, char) {
   return char.repeat(text.length);
}


function contamination(text, char) {
   return text.split("").fill(char).join("");
}


function contamination(text, char) {
   return text.replace(/./g, char);
}


console.log(contamination("abc", "z"), "zzz");
console.log(contamination("", "z"), "");
console.log(contamination("abc", ""), "");
console.log(contamination("_3ebzgh4", "&"), "&&&&&&&&");
console.log(contamination("//case", " "), "      ");

File: Python/8py 1/contamination.py
"""
|--------------------------------------------------------------------------
| Contamination #1 -String-
|--------------------------------------------------------------------------
|
"""

def contamination(text, char):
   return char * len(text)


def contamination(text, char):
   return "".join(char for x in text)


from re import sub
def contamination(text, char):
   return sub(".", char, text)


print(contamination("abc","z"), "zzz")
print(contamination("","z"), "")
print(contamination("abc",""), "")
print(contamination("_3ebzgh4","&"), "&&&&&&&&")
print(contamination("//case"," "), "      ")